src/gauge.js

```diff
@@ -33,6 +33,30 @@ class Gauge {
     return this.value_;
   }
 
+  /**
+   * Get the current value.
+   *
+   * @deprecated Use get() instead.
+   *
+   * @return {number} The current value for this gauge.
+   */
+  value() {
+    return this.value_;
+  }
+
+  /**
+   *
+   * @deprecated Use set instead.
```

Comment:
This comment looks odd compared to the others with the deprecation annotation before the description.

Reply:
Thanks, fixed.